Core/DISET/private/BaseClient.py

```diff
@@ -252,9 +254,10 @@ def _connect( self ):
       if not retVal[ 'OK' ]:
         if self.__retry < 5:
           url = "%s://%s:%d/%s" % ( self.__URLTuple[0], self.__URLTuple[1], int( self.__URLTuple[2] ), self.__URLTuple[3] )
-          if url not in self.__bannedUrls: 
-            gLogger.notice( "URL banned", "%s" % url )
-            self.__bannedUrls += [url]   
+          if url not in self.__bannedUrls:
+            self.__bannedUrls += [url]
+            if len( self.__bannedUrls ) < self.numberOfURLs:
```

Comment:
No. Some services can have several alternative URLs. If one of the URLs is not responding at the first attempt, it is removed from the list of URLs and the message is printed to warn that it will not be further tried. However, if after the removal the URL list becomes empty ( e.g. was only one URL in the list ), then the whole list is restored for further retrials. Effectively, nothing is banned in this case and that's why I forbid to print this warning message.